JS/Lesson32/onur/my-app/src/components/RecipeDetail/index.jsx

```diff
@@ -0,0 +1,12 @@
+import './styles.css'
```

Comment:
I'd add a close button to the RecipeDetail component. If it's clicked, you can update `selectedRecipe`in the `RecipeList` to not display any recipe details.